glide-core/src/protobuf/redis_request.proto

```diff
@@ -261,6 +261,13 @@ message Command {
     }
 }
 
+// Used for script requests with large keys or args vectors
+message ScriptInvocationPointers {
+    string hash = 1;
+    optional uint64 keys_pointer = 2;
```

Comment:
why not to store this in `ScriptInvocation`?
```proto
message ScriptInvocation {
    string hash = 1;
    oneof keys {
        repeated bytes keys_as_strings = 2;
        uint64 keys_pointer = 3;
    }
    oneof args {
        repeated bytes args_as_strings = 4;
        uint64 args_pointer = 5;
    }
}
```
or something like
```proto
message ScriptInvocation {
    string hash = 1;
    oneof params {
        message ScriptStrings {
            repeated bytes keys_as_strings = 1;
            repeated bytes args_as_strings = 2;
        } = 2
        message ScriptPointers {
            uint64 keys_pointer = 1;
            uint64 args_pointer = 2;
        } = 3;
    }
}
```

Reply:
probably to avoid breaking Node/Java script invocation calls?

Reply:
@Yury-Fridlyand read the description of the PR...